grids.md

```diff
@@ -0,0 +1,28 @@
+# CSS Grids
+
+A two-dimensional structure made up of a series of intersecting vertical and horizontal axes used to structure content. The grid serves as a framework on which a designer can organize text and images in a rational, easy to absorb manner.
+
+## Beginning Grids
+
+Before starting CSS Grids, follow the [Visual Principles](visual-principles.md)
+trail map.
+
+* Read [Grid Systems: Principles of Organizing Type](http://amzn.to/grid-systems-principles)
+* Read [Grid Systems in Graphic Design](http://amzn.to/grid-systems-graphic-design)
+* Read [Mark Boulton's Designing Grid Systems](http://goo.gl/VOGP)
```

Comment:
Expand to: http://www.markboulton.co.uk/journal/five-simple-steps-to-designing-grid-